### DISCUSSION TITLE
Backdrop support

### DISCUSSION THREAD

**Discussion by Fusseldieb** [home-assistant/frontend#10014] [Dashboard]
![image](https://user-images.githubusercontent.com/4715129/132937533-a5ba9ac8-5bdb-410d-a1ad-1f4b840d4ece.png)

A lot of themes would benefit of the relatively new backdrop property, since it blurs the background of cards and stuff.
For whatever reason, however, it was just closed.
I vote to reopen https://github.com/home-assistant/frontend/pull/7737

**Comment by ncodee** [comment]
+1. Please reconsider adding this.

**Comment by matthiasdebaat** [comment]
This has recently been added https://github.com/home-assistant/frontend/pull/20447
